### DISCUSSION TITLE
how to activate persistent sync without logged in user

### DISCUSSION THREAD

**Discussion by PacoDavid** [abraunegg/onedrive#1467] [Q&A] [answered]
hello all I use this app, and functionally it's ok, but when my server close the session the sync stopped, and is need login in the server to reactive sync.

how can run service even if my user are disconected?

**Comment by abraunegg** [comment, marked as answer]
@PacoDavid 
Please refer to the documentation: https://github.com/abraunegg/onedrive/blob/master/docs/USAGE.md#running-onedrive-as-a-system-service

**Reply by PacoDavid** [reply]
@abraunegg 

I checked and i makes changes necessaries, now is functional and work as is required, thanks
